src/main/java/io/debezium/connector/yugabytedb/YBPartition.java

```diff
@@ -42,6 +43,11 @@ public YBPartition(String tableId, String tabletId, boolean isTableColocated) {
         this.colocated = isTableColocated;
     }
 
+    public static YBPartition from(String fullPartitionId) {
+        String[] tableTablet = fullPartitionId.split("\\.");
+        return new YBPartition(tableTablet[0], tableTablet[1], true);
```

Comment:
This currently assumed that the partition ID in the topic will always be the fullPartitionId but that will not be the case if connector is in streaming mode.